### PR TITLE
add a note about double and float data types precision

### PR DESCRIPTION
**Pull request by kovrus**
https://www.postgresql.org/docs/9.2/static/datatype-numeric.html#DATATYPE-FLOAT

### CODE REVIEW

File: blackbox/docs/sql/data_types.txt [hunk 1]
```diff
@@ -96,11 +96,20 @@ Columns of type string can also be analyzed. See :ref:`sql_ddl_index_fulltext`.
...
-Example::
+Crate supports a set of numeric types: ``integer``, ``long``, ``short``,
+``double``, ``float`` and ``byte``.
+
+The ``float`` and ``double`` data types are inexact, variable-precision
+numeric types. It means that these types are stored as an approximation.
+Therefore, storage, calculation, and retrieval of the value will not always
+result in an exact representation of the actual floating-point value.
+For instance, the result of applying ``sum`` or ``avg`` aggregate functions
+may slightly vary from run to run or comparing floating-point values for
```
Comment (matriv) [inline blackbox/docs/sql/data_types.txt]: I would rephrase: "from run to run" to "between query executions" or "between executions"

Comment (kovrus) [inline blackbox/docs/sql/data_types.txt]: rephrased
